morula.yml

```diff
@@ -1,5 +1,5 @@
 before-all: "exorelay-javascript"
 after-all: ""
 always: ""
-never: ""
+never: "bin"
```

Comment:
Not sure if we should add `bin` here, or write dummy `bin/spec`, `bin/setup`, and `bin/spec_ci` scripts to the `bin` folder. I wasn't sure if it was conventional to have a `bin/bin` folder...